man/import.q.sorts.Rd

```diff
@@ -1,14 +1,15 @@
 \name{import.q.sorts}
 \alias{import.q.sorts}
-\title{Q methodology: imports q sorts from csvs}
+\title{Q methodology: imports q sorts from CSV}
 \description{
-	Turns raw q-sorts (from csv) into clean, veriefied q.sorts array or matrix.
+	Turns raw Q sorts (from .CSV) into clean, verified q.sorts array or matrix.
 }
```

Comment:
done in https://github.com/maxheld83/qmethod/commit/dec047f6fc92338f5eb5a4f4221cf046d323e6e6